### DISCUSSION TITLE
How to access imported bookmarks from the web site?

### DISCUSSION THREAD

**Discussion by l0b0** [go-shiori/shiori#749] [Q&A] [answered]
The shiori web interface doesn't seem to have any import functionality, and bookmarks imported via the CLI are not visible in the web interface. How do I import such that bookmarks are visible in the web interface?

**Comment by Monirzadeh** [comment]
can you provide a sample file from wallabag?

**Comment by fmartingr** [comment, marked as answer]
What I think it happened to you is that you used the CLI to import bookmarks and go stored in a database file that is not the same your server is using. If you point the `SHIORI_DIR` environment variable to where your server database is, the bookmarks should go into the server database and you should see them there. (Create a backup first!)
